config/examples/segment.yml

```diff
@@ -1,5 +1,5 @@
 base: &default
-  enabled: true
+  enabled: false
```

Comment:
What is this segment thing for?

Reply:
There was some discussion behind it. I forgot though.